src/main/java/com/palantir/docker/compose/execution/GracefulShutdownStrategy.java

```diff
@@ -18,11 +19,11 @@
     private static final Logger log = LoggerFactory.getLogger(GracefulShutdownStrategy.class);
 
     @Override
-    public void shutdown(DockerCompose dockerCompose) throws IOException, InterruptedException {
+    public void shutdown(DockerComposeRule rule) throws IOException, InterruptedException {
         log.debug("Killing docker-compose cluster");
-        dockerCompose.down();
-        dockerCompose.kill();
-        dockerCompose.rm();
+        rule.dockerCompose().down();
```

Comment:
It means that any implementations of the `ShutdownStrategy` interface have maximal functionality available to them